components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -46,28 +48,30 @@ def index_with_filter
 
         @filtered_applications = value[:filtered_applications]
         @recent_determined_hbx_id = value[:recent_determined_hbx_id]
+
+        respond_to do |format|
+          format.html
+        end
       else
-        render json: result.failure.to_h, status: 422
+        respond_to do |format|
+          format.json { render json: result.failure.to_h, status: 422 }
+        end
       end
     end
 
-    def new
```

Comment:
Why did we take out this?

Reply:
we are not using these end points in FAA applications controller
- new
- help paying for coverage